modules/client-configuration/pages/supported-features-oracle.adoc

```diff
@@ -0,0 +1,67 @@
+[[supported-features-oracle]]
+= Supported Oracle Features
+
+
+This table lists the availability of various features on {oracle} clients.
+
+The icons in this table indicate:
+
+* icon:check[role="green"] the feature is available on both traditional and Salt clients
+* icon:times[role="danger"] the feature is not available
+* icon:question[role="gray"] the feature is under consideration, and may or may not be made available at a later date
+* ``Traditional`` the feature is supported only on traditional clients
+* ``Salt`` the feature is supported only on Salt clients
+
+
+[cols="1,1,1,1", options="header"]
+.Supported Features on {oracle} Operating Systems
+|===
+| Feature | {oracle}{nbsp}6 | {oracle}{nbsp}7 | {oracle}{nbsp}8
+| Client | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Operating system packages  | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Registration | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Install packages | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Apply patches (requires CVE ID) | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Remote commands | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| System package states | ``Salt`` | ``Salt`` | ``Salt``
+| System custom states | ``Salt`` | ``Salt`` | ``Salt``
+| Group custom states | ``Salt`` | ``Salt`` | ``Salt``
+| Organization custom states    | ``Salt`` | ``Salt`` | ``Salt``
+| System set manager (SSM) | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| Service pack migration | N/A | N/A | N/A
+| Basic Virtual Guest Management icon:asterisk[role="none"] | ``Traditional`` | icon:check[role="green"] | ``Salt``
+| Advanced Virtual Guest Management icon:asterisk[role="none"] | icon:times[role="danger"] | ``Salt`` | ``Salt``
+| Virtual Guest Installation (Kickstart), as Host OS | ``Traditional`` | ``Traditional`` | icon:times[role="danger"]
+| Virtual Guest Installation (image template), as Host OS | ``Traditional`` | icon:check[role="green"] | ``Salt``
+| System deployment (PXE/Kickstart) | icon:check[role="green"] | icon:check[role="green"] | ``Salt``
+| System redeployment (Kickstart) | ``Traditional`` | icon:check[role="green"] | ``Salt``
+| Contact methods | ``Traditional``: OSAD, RHNSD, SSH-push. +
```

Comment:
Is the "+" at the end of the line actually needed here?  Wonders of adoc?

Reply:
no, i think that's copypasta.